Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,12 @@
+class ContactsPhonetic < Formula
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/7f13963b6948e27dca4ae06ba9939ffe26cf0089.tar.gz"
+  version "0.1.1"
```

Comment:
Tried. It said the version in the URL is redundant, which made no sense to me. So I rebased it and pushed again.